common/persistence/class.PhpFileDriver.php

```diff
@@ -194,6 +192,29 @@ private function writeFile($id, $value, $preWriteValueProcessor = null)
         }
     }
 
+    /**
+     * Create directory and suppress warning message
+     * @param $dirpath
+     * @param int $mode
+     * @return bool
+     */
+    private function makedir(string $dirpath, int $mode)
```

Comment:
```suggestion
    private function makeDirectory(string $path, int $mode)
```
Inside the method, it is clear that it is a directory path.